### PR TITLE
chore(angular): bump ngrx to 13.0.0

### PR DESCRIPTION
**Pull request by Coly010**
Bump NgRx to v13

### CODE REVIEW

File: packages/angular/migrations.json [hunk 1]
```diff
@@ -1074,6 +1074,34 @@
...
+          "version": "~13.0.0-beta.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@ngrx/store-devtools": {
+          "version": "~13.0.0-beta.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@ngrx/component-store": {
+          "version": "~13.0.0-beta.0",
+          "alwaysAddToPackageJson": false
```
Comment (leosvelperez) [inline packages/angular/migrations.json:1104]: Version should be `~13.0.0`.

File: package.json [hunk 2]
```diff
@@ -54,13 +54,13 @@
...
-    "@ngrx/schematics": "12.5.1",
-    "@ngrx/store": "12.5.1",
-    "@ngrx/store-devtools": "12.5.1",
+    "@ngrx/component-store": "13.0.0",
+    "@ngrx/effects": "13.0.0",
+    "@ngrx/entity": "13.0.0",
+    "@ngrx/router-store": "13.0.0",
+    "@ngrx/schematics": "13.0.0",
+    "@ngrx/store": "13.0.0",
+    "@ngrx/store-devtools": "13.0.0",
```
Comment (leosvelperez) [inline package.json:63]: We should be good using `~13.0.0`.

**Review by leosvelperez — CHANGES_REQUESTED** [review]

File: packages/jest/plugins/resolver.ts [hunk 3]
```diff
@@ -60,6 +61,13 @@ module.exports = function (path: string, options: ResolveOptions) {
           ...pkg,
           main: pkg.main || pkg.es2015 || pkg.module,
         }),
+        pathFilter: (pkg) => {
+          if (!pkg.exports) {
+            return path;
+          }
+
+          return resolveExports(pkg, path) || path;
+        },
```
Comment (Coly010) [inline packages/jest/plugins/resolver.ts:70]: @barbados-clemens 
I had to add a path filter to the Jest resolver so that it could find the new `exports` field from ESM package.json files.
Could you check that I haven't overlooked anything here or missed any potential edge cases etc?

Comment (barbados-clemens) [reply packages/jest/plugins/resolver.ts:70]: I ran through a handful of tests/checks and everything ran as expected. @Cammisuli said he'll take a look when he gets a chance as well.

**Review by barbados-clemens — APPROVED** [review]
Jest looks good from my view. Jon is going to take a look as well just to be safe.

**Review by brandonroberts — APPROVED** [review]

**Review by leosvelperez — APPROVED** [review]
